HBase 2 minicluster tests fail

We have a test that ensures compatibility with HBase that runs a local HBase cluster.  Our HBase 2 tests currently fail

Here's how you can run the test 
`mvn clean verify -PhbaseLocalMiniClusterTestH2`

```
unning com.google.cloud.bigtable.hbase.IntegrationTests
2017-12-07 22:36:18 INFO  MiniClusterEnv:34 - Starting hbase minicluster
2017-12-07 22:36:19 ERROR ZooKeeperServer:472 - ZKShutdownHandler is not registered, so ZooKeeper server won't take any action on ERROR or SHUTDOWN server state changes
2017-12-07 22:36:19 ERROR HRegionServer:657 - Failed construction RegionServer
java.lang.UnsatisfiedLinkError: failed to load the required native library
	at org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.Epoll.ensureAvailability(Epoll.java:78)
	at org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.EpollEventLoopGroup.<clinit>(EpollEventLoopGroup.java:38)
	at org.apache.hadoop.hbase.util.NettyEventLoopGroupConfig.<init>(NettyEventLoopGroupConfig.java:61)
	at org.apache.hadoop.hbase.regionserver.HRegionServer.setupNetty(HRegionServer.java:679)
	at org.apache.hadoop.hbase.regionserver.HRegionServer.<init>(HRegionServer.java:548)
	at org.apache.hadoop.hbase.master.HMaster.<init>(HMaster.java:471)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.apache.hadoop.hbase.util.JVMClusterUtil.createMasterThread(JVMClusterUtil.java:131)
	at org.apache.hadoop.hbase.LocalHBaseCluster.addMaster(LocalHBaseCluster.java:205)
	at org.apache.hadoop.hbase.LocalHBaseCluster.<init>(LocalHBaseCluster.java:152)
	at org.apache.hadoop.hbase.MiniHBaseCluster.init(MiniHBaseCluster.java:214)
	at org.apache.hadoop.hbase.MiniHBaseCluster.<init>(MiniHBaseCluster.java:94)
	at org.apache.hadoop.hbase.HBaseTestingUtility.startMiniHBaseCluster(HBaseTestingUtility.java:1128)
	at org.apache.hadoop.hbase.HBaseTestingUtility.startMiniCluster(HBaseTestingUtility.java:1082)
	at org.apache.hadoop.hbase.HBaseTestingUtility.startMiniCluster(HBaseTestingUtility.java:953)
	at org.apache.hadoop.hbase.HBaseTestingUtility.startMiniCluster(HBaseTestingUtility.java:947)
	at org.apache.hadoop.hbase.HBaseTestingUtility.startMiniCluster(HBaseTestingUtility.java:876)
	at com.google.cloud.bigtable.hbase.test_env.MiniClusterEnv.setup(MiniClusterEnv.java:37)
	at com.google.cloud.bigtable.hbase.test_env.SharedTestEnv.retain(SharedTestEnv.java:56)
	at com.google.cloud.bigtable.hbase.test_env.SharedTestEnv.get(SharedTestEnv.java:38)
	at com.google.cloud.bigtable.hbase.test_env.SharedTestEnvRule.before(SharedTestEnvRule.java:53)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:46)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.NativeStaticallyReferencedJniMethods.epollin()I
	at org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.NativeStaticallyReferencedJniMethods.epollin(Native Method)
	at org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.Native.<clinit>(Native.java:66)
	at org.apache.hadoop.hbase.shaded.io.netty.channel.epoll.Epoll.<clinit>(Epoll.java:33)
```

--- a/bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/test_env/MiniClusterEnv.java
+++ b/bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/test_env/MiniClusterEnv.java
@@ -33,6 +33,9 @@ class MiniClusterEnv extends SharedTestEnv {
   protected void setup() throws Exception {
     LOG.info("Starting hbase minicluster");
 
+    System.setProperty("org.apache.hadoop.hbase.shaded.io.netty.packagePrefix",
+      "org.apache.hadoop.hbase.shaded.");
+
     helper = HBaseTestingUtility.createLocalHTU();
     helper.startMiniCluster();
 

